apps/stats-dapp/src/containers/ProposalDetail/ProposalDetail.tsx

```diff
@@ -172,9 +174,10 @@ export const ProposalDetail = () => {
             label="chain:"
             value={
               <span className="flex items-center p-2 space-x-2">
-                <TokenIcon name={chain} size="lg" />
-                <Typography variant="body1" className="block uppercase">
-                  {chain}
+                <TokenIcon name={mapChainIdToLogo(Number(chain))} size="lg" />
```

Comment:
Consider using `ChainIcon` instead of `TokenIcon` here, the `ChainIcon` will render based on the chain name. @devpavan04